core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/LDAPSearchSpecification.java

```diff
@@ -42,6 +51,7 @@ public class LDAPSearchSpecification {
     private static final String EQUALS_SIGN = "=";
     private static final String SERVICE_NAME_ATTRIBUTE = "sn";
     private static final String VALUE_SEPARATOR = ",";
+    private final String ldapTimestampFormat;
```

Comment:
```suggestion
    private String ldapTimestampFormat = DEFAULT_LDAP_BASIC_TIMESTAMP_FORMAT;
```

Reply:
Then we may not required to update the existing constructor.

Reply:
This would be a redundant initializer. Since in the constructor we anyway initialize this.

Reply:
Not redundant actually. It will be seen as redundant as the object creator is you in this case. But somebody can create an object with other constructor also. We don't need to have that logic inside that constructor.

Reply:
At this point that is the expected behaviour right ? To let customize the timestamp format